src/index.html

```diff
@@ -11,6 +11,37 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <!-- <h1>Search bar airbnb</h1> -->
+    <div class="search-bar-big">
+      <div class="search-container-big">
+        <form id="big" 
+        action="https://localhost:3000/api" 
+        method="post" >
+          <div class="search-image">
```

Comment:
same here. you can set this class to the label

Reply:
did the same